src/QUESTIONS.md

```diff
@@ -246,6 +269,17 @@ You can list all available software on CheriBSD with `pkg64c rquery %n` and
 Alternatively, you can search for a specific package using
 `pkg64c search <pattern>` and `pkg64 search <pattern>`.
 
+If you simply require a list of packages independently of CheriBSD, there are browsable HTML lists of the ports for both architectures: [AArch64](https://pkg.cheribsd.org/CheriBSD:20220828:aarch64.html) (hybrid) and [AArch64c](https://pkg.cheribsd.org/CheriBSD:20220828:aarch64c.html) (CHERI). In addition, there are tarballs on [pkg.cheribsd.org](https://pkg.cheribsd.org) that contain the raw JSON-formatted package data. To generate a list of CHERI packages with specific fields, you can use `curl`, `tar`, and `jq`:
+```bash
+curl -O https://pkg.cheribsd.org/CheriBSD:20220828:aarch64c/packagesite.txz
+tar xf *.txz
+jq -j '.name," (",.version,"; ",.abi,"): ",.www,"\n"' *.yaml | sort -uf > aarch64c.txt
+```
+To create a similar list of hybrid packages, use the following instead:
+```bash
+curl -O https://pkg.cheribsd.org/CheriBSD:20220828:aarch64/packagesite.txz
+```
+
```

Comment:
This seems to be redundant since we have the websites listing packages. Is there any use case for this that I'm missing?

Reply:
I would favour using JSON for CI, monitoring, and continuous testing over raw HTML.

Reply:
Why would you want to monitor the list of packages? A package repository includes a stable list of packages that rarely changes. We've updated only toolchain a couple of times.

My concern is that this answer got quite lengthy while the `jq` example is not covering a case we've seen in the ecosystem. It might be better to include it in a separate Q&A once it appears.

Reply:
I've no objection to removing the code example and reserving it for possible future use.